tests/integration/components/select-list-test.js

```diff
@@ -183,3 +183,15 @@ test('binds properly to `title` attribute', function(assert) {
 
   assert.equal(this.$('select').attr('title'), title);
 });
+
+test('remains invalid when prompt is selected', function(assert) {
+  var options = [];
+  this.set('options', options);
+  this.set('prompt', 'First prompt');
+
+  this.render(hbs`
+    {{select-list content=options value=selection prompt=prompt}}
+  `);
+
+  assert.equal(this.$('select option:selected')[0].validity.valid, false);
```

Comment:
Not good enough; ``validity`` is what the browser uses to determine whether the form is valid.  Especially when you only have 1 element in your form, and that is a ``select``, this makes the difference between the form submitting and not.  Simply testing for ``value=""`` isn't enough; we need to test that the ``<select>`` element is invalid.